client/src/pages/cejst.tsx

```diff
@@ -14,7 +13,7 @@ interface IMapPageProps {
 }
 
 const CEJSTPage = ({location}: IMapPageProps) => {
-  const [features, setFeatures] = useState<Feature<Geometry>[]>([]);
+  const [features] = useState<Feature<Geometry>[]>([]);
```

Comment:
Humm.. `useState` without a setter? I think that just makes it a `const`?
Is this just a temporary thing so you can easily remove the `<MapControls>`

Reply:
This is temporary for now, yes - we might need it later

Reply:
OK. Maybe a comment explaining that and then the PR LGTM.

Reply:
Good call - added comment just now